### ISSUE TITLE
Domains' labels during training the feature extractor

### ISSUE THREAD

**Issue by phquang** [belaalb/TI-DG#3]
Thanks for the interesting work.
I want to ask why the labels in the domain discriminators' losses were flipped during training the feature extractor.
https://github.com/belaalb/G2DM/blob/e48523722bc77304c204bba78893fb58605e3453/pacs-ours/train_loop.py#L196-L198
Isn't it ``curr_y_domain = torch.where(y_domain == i, torch.ones(y_domain.size(0)), torch.zeros(y_domain.size(0)))`` as it was during training the discriminators?

**Comment by belaalb** [comment]
Hi! If we flip the labels and the sign of the loss we can show we have a zero-sum minimax game.

**Comment by belaalb** [comment]
Hi! If we flip the labels to have a game similar to a non-saturating GAN.
